llvm/tools/llvm-cas-dump/MCCASPrinter.h

```diff
@@ -1,47 +1,54 @@
-//===----------------------------------------------------------------------===//
+//===-----------------------------------------------------------*- C++ -*--===//
 //
 // Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
 
+#ifndef LLVM_TOOLS_LLVM_CAS_DUMP_MCCASPRINTER_H
```

Comment:
oops, good catch!

Reply:
I also added some missing plumbing for llvm-cas-dump to LIT.